crates/autopilot/src/infra/persistence/dto/quote.rs

```diff
@@ -0,0 +1,21 @@
+use {
+    crate::{boundary, domain},
+    number::conversions::big_decimal_to_u256,
+    primitive_types::H160,
+};
+
+pub fn into_domain(quote: boundary::DatabaseQuote) -> Result<domain::Quote, InvalidConversion> {
```

Comment:
I'm a bit surprised to see `dto` in the path here.
My understanding is that `dto`s are only dumb data objects that can be (de)serialized to send them to other services via HTTP requests or some other means. In this case it seems more appropriate to me to have these functions as part of the module that implements the operations on the DB.

Reply:
`boundary::DatabaseQuote` is kind of a `dto` object between autopilot and database, since it's a serializable/deserializable representation of the `domain::Quote`.

I wanted to have this way of structuring paths so I can refer to `to_domain/from_domain` methods as `dto::object::to_domain()` for example, which reads much more nicely than `object::dto::to_domain()` at least IMO